CHANGELOG.md

```diff
@@ -4,6 +4,43 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.14](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.14) - _TODO:Add actual release date_
+
+Zebra's latest alpha continues our work on NU5, including Orchard and Transaction V5.
+
+### Added
+
+- Reject UTXO double spends (#2511)
+- Add a ValueBalance type (#2505)
+- Implement Sum for Amount (#2500)
+- Add an OrderedUtxo type for transparent spend validation (#2502)
+- Pass the finalized state to chain contextual validation (#2503)
+- Calculate incremental note commitment trees (#2407)
+
+#### Documentation
+
+- Add value pools design to book summary (#2520)
+
+#### Testing
+
+- Run `rustfmt` inside nullifier and utxo property tests (#2515)
+- Test consensus-critical Amount deserialization (#2487)
+- Add more details to test coverage increase entry (#2499)
```

Comment:
I'm not sure fixing the previous changelog is that interesting either. But if it is, we should say that we edited the previous changelog.

Reply:
ah, I should have looked into the PR to see what it did. FWIW, I don't think it's very interesting either and I have removed it from the PR